components/sidenav/metadata/sidenav.yml

```diff
@@ -1,135 +1,220 @@
-name: Side Navigation
+name: Side navigation
```

Comment:
All of the other components on the sidebar of the docs site are capitalized currently. It would be odd to only change one. There is a mix of capitalized words and not capitalized words in Storybook. I do see in some of the guidelines links that they don't capitalize the second word (which seems odd to me, looks like a mistake). @castastrophe thoughts on which to prefer to make this consistent across docs and Storybook?

Reply:
I think it might be related to this design guidance - https://spectrum.adobe.com/page/grammar-and-mechanics/#Capitalization

"Use sentence case for all aspects of designing Adobe product experiences, including titles and UI elements (e.g., tooltips, tabs, menu items)."

Reply:
for documentation, chatted about this in slack https://adobedesign.slack.com/archives/G019JTYMT6H/p1686755962486599 

yes we should be using sentence casing as it follows the style guide